From: David Hildenbrand <david@redhat.com>
Subject: arch/loongarch/include/asm/tlb.h: remove __tlb_remove_tlb_entry()
Date: Tue, 16 Apr 2024 15:51:30 +0200

In preparation for enabling -Wextra, remove this and rely upon the
asm-generic version.

Fixes this warning:

    ./include/asm-generic/tlb.h:629:10: error: parameter 'ptep' set
but not used [-Werror,-Wunused-but-set-parameter]
      629 |                 pte_t *ptep, unsigned int nr, unsigned long address)
          |                        ^


Link: https://lkml.kernel.org/r/c82af143-b620-44d9-8647-f52096b851ab@redhat.com
Signed-off-by: David Hildenbrand <david@redhat.com>
Reported-by: Miguel Ojeda <miguel.ojeda.sandonis@gmail.com>
Tested-by: Arnd Bergmann <arnd@arndb.de>
Cc: Huacai Chen <chenhuacai@kernel.org>
Cc: Ryan Roberts <ryan.roberts@arm.com>
Cc: WANG Xuerui <kernel@xen0n.name>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 arch/loongarch/include/asm/tlb.h |    2 --
 1 file changed, 2 deletions(-)

--- a/arch/loongarch/include/asm/tlb.h~include-asm-generic-tlbh-629-10-error-parameter-ptep-set-but-not-used
+++ a/arch/loongarch/include/asm/tlb.h
@@ -132,8 +132,6 @@ static __always_inline void invtlb_all(u
 		);
 }
 
-#define __tlb_remove_tlb_entry(tlb, ptep, address) do { } while (0)
-
 static void tlb_flush(struct mmu_gather *tlb);
 
 #define tlb_flush tlb_flush
_
